bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -260,6 +267,33 @@ impl Room {
         Ok(Timeline::new(timeline))
     }
 
+    pub async fn media_events_timeline(
+        &self,
+        filter: MediaEventsTimelineFilter,
+    ) -> Result<Arc<Timeline>, ClientError> {
+        let room = &self.inner;
+
+        let mut builder = matrix_sdk_ui::timeline::Timeline::builder(room);
```

Comment:
Could we also ask for a timeline internal id prefix in this function's signature, and pass it here? This would help tracking issues in rageshakes, since the internal id prefix is used to distinguish across different timeline instances.

Reply:
We may want to add a note that we're not passing a UTD hook here, but that's fine because we should be flagging those in the main timeline anyways.